src/20_Components/amp-sidebar.html

```diff
@@ -56,7 +56,7 @@
 
     `amp-fit-text` expands or shrinks its font size to fit the content within the space given to in the menu.
   -->
-    <amp-sidebar id='sidebar1' layout="nodisplay">
+    <amp-sidebar id='sidebar1' layout="nodisplay" side="right">
```

Comment:
we need a different name when we have two in the page, I will change the menu one and leave this one as sidebar.